tunnel_virtual_networks.go

```diff
@@ -0,0 +1,157 @@
+package cloudflare
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+// TunnelVirtualNetwork is segregation of Tunnel IP Routes via Virtualized Networks to handle overlapping private IPs in your origins.
+type TunnelVirtualNetwork struct {
+	ID               string     `json:"id"`
+	Name             string     `json:"name"`
+	IsDefaultNetwork bool       `json:"is_default_network"`
+	Comment          string     `json:"comment"`
+	CreatedAt        *time.Time `json:"created_at"`
+	DeletedAt        *time.Time `json:"deleted_at"`
+}
+
+type TunnelVirtualNetworksListParams struct {
+	AccountID string
```

Comment:
this needs to be `url:"-"` if you do not want it to appear in the query params as `AccountID=`.